modules/Slash/Information/gpt.go

```diff
@@ -67,3 +100,21 @@ func getResponse(ctx context.Context, client gpt3.Client, question string) (resp
 
 	return response, nil
 }
+
+func moderationTest(str string, pResp *profanityStruct) (bool, string) {
+	for _, word := range strings.Split(str, " ") {
+		if word[0] == '@' {
```

Comment:
Shouldn't this be `<@` instead? Pings look like `<@userid>` and `<@&roleid>`

Reply:
Those are placeholders, after the text gets formatted for sending it will be in the form of `@` rather than `<@`. Then just before we send, we run moderation and check for that.